nni/experiment/config/experiment_config.py

```diff
@@ -61,6 +61,7 @@ class ExperimentConfig(ConfigBase):
     # In latter case hybrid training services can have different settings.
 
     experiment_name: Optional[str] = None
+    experiment_type: str = 'hpo'
```

Comment:
The type hint should be `Literal['hpo']` (from typing_extensions import).

Reply:
fixed